apps/tup-cms/src/taccsite_cms/templates/snippets/css-news-hacks.html

```diff
@@ -1,15 +0,0 @@
-<style class="css-news-hacks">
-    /* To hide byline of an article by another outlet */
-    /* NOTE: Requires author add "external" tag to the article */
-    /* TODO: Figure out how to show external author that user enters */
-    .has-blog-tag-external .byline {
-        display: none;
-    }
-
-    /* To hide the "by: " in the byline */
-    /* CAVEAT: The &nbsp; between "by:" and author name is still visible */
-    /* TODO: (TUP-552) Whether to hide this in Core, TUP, or any CMS */
-    .byline span {
-        display: none;
-    }
```

Comment:
This code is already present and taking effect because it was [migrated to Core-CMS styles](https://github.com/TACC/Core-CMS/pull/773/files#diff-80e9452c1331968133e13bc20494a63929fa1d2221c9f9bc580ead19fab33f6eR92-R95) 2 months ago.

Reply:
| "by" is hidden via CMS code<br/><sup>(as on https://tacc.utexas.edu/news/latest-news/)</sup> |
| - |
| <img width="1349" alt="styles migrated to cms" src="https://github.com/TACC/tup-ui/assets/62723358/e88292e1-9b88-4ffa-b02b-5108e529bbfe"> |